pkg/restapi/vc/operation/operations_test.go

```diff
@@ -525,3 +723,27 @@ func (r *mockReader) Read(p []byte) (n int, err error) {
 func (r *mockReader) Close() error {
```

Comment:
(optional) I feel like it would be more clear if the mockReader was named so that it's obvious that it always fails. Just my $0.02

Reply:
i think mock reader is fine for now, unless you have better name lol

Reply:
My suggestion is "failingReader". But if you still prefer mockReader then that's ok